app/services/request_logger.rb

```diff
@@ -34,6 +39,15 @@ def log_request(duration, payload)
       )
     end
 
+    def updated_remarks(remarks)
+      remarks.map { |key, value|
```

Comment:
This is more idiomatically done with transform_values?

Reply:
@starswan I need to filter by key first and transform_values doesn't allow me to filter conditionally